plugin/output/splunk/splunk.go

```diff
@@ -208,8 +212,12 @@ func (p *Plugin) send(data []byte) error {
 		return fmt.Errorf("can't decode response: %w", err)
 	}
 
-	code := root.Dig("code").AsInt()
-	if code > 0 {
+	code := root.Dig("code")
```

Comment:
AsInt() returns 0 if code if nil.

Reply:
yes. that is why I made this PR)

Reply:
there is difference between 0 error code and missing "code" field